spec/rspec/matchers/built_in/satisfy_spec.rb

```diff
@@ -39,4 +63,19 @@
       expect(true).not_to satisfy { |val| val }
     }.to fail_with("expected true not to satisfy block")
   end
+
+  context "when a custom description is provided" do
+    it "passes if block returns false" do
+      expect(false).not_to satisfy("be true") { |val| val }
+      expect(false).not_to satisfy("be true") do |val|
+        val
+      end
+    end
+
+    it "fails if block returns true" do
```

Comment:
Same here.  I'd suggest: `it "fails with the custom description if block returns true"`.